test_files/enum.sickle.ts

```diff
@@ -13,4 +13,19 @@ export enum EnumTest2 {XYZ, PI = 3.14159}
 (<any>EnumTest2).XYZ = 0;
 /** @type {EnumTest2} */
 (<any>EnumTest2).PI =  3.14159;
+/** @typedef {number} */
```

Comment:
This comment is a docstring on the below enum.  I bet it's not applying right.

To fix this, you need to emit the enum with no preceding comments.